src/figure/formation.c

```diff
@@ -186,10 +189,84 @@ void formation_record_fight(formation *m)
     m->recent_fight = 6;
 }
 
+int is_formation_halted(const formation *m)
```

Comment:
```suggestion
static int is_formation_halted(const formation *m)
```

`static` keeps the function internal to this file.